boxes/react/package.json

```diff
@@ -11,10 +11,12 @@
     "prep": "yarn clean && yarn compile && yarn codegen",
     "dev": "yarn prep && webpack serve --mode development",
     "build": "yarn prep && webpack",
-    "serve": "serve -p 3000 ./dist",
+    "serve": "webpack serve --no-open --mode development",
     "formatting": "prettier --check ./src && eslint ./src",
     "formatting:fix": "prettier -w ./src",
-    "test": "NODE_NO_WARNINGS=1 node --experimental-vm-modules $(yarn bin jest) --runInBand"
+    "test": "yarn test:node && yarn test:browser",
+    "test:node": "NODE_NO_WARNINGS=1 node --experimental-vm-modules $(yarn bin jest) --runInBand",
+    "test:browser": "npx playwright test"
```

Comment:
If we're adding playwright tests, let's add it as an explicit devdependency, so we can get the version pinned

Reply:
Makes sense, I also need to pin it explicitly in the dockerfile, will do that too